utest/variables/test_variables.py

```diff
@@ -222,7 +226,7 @@ def test_math_with_internal_vars_does_not_work_if_first_var_is_float(self):
         assert_raises(VariableError, self.varz.replace_scalar, '${${1.1} * ${2}}')
         assert_raises(VariableError, self.varz.replace_scalar, '${${1.1}/${2}}')
 
-    def test_list_variable_as_scalar(self):
+    def var_variable_as_scalar(self):
```

Comment:
This looks like an accidental change.

Reply:
Reverted by 312360d873a4ab47b366f79fe9bf9a288f47fdea